custom_components/hacs/websocket/__init__.py

```diff
@@ -111,7 +111,6 @@ async def hacs_info(
                 "debug": hacs.configuration.debug,
                 "dev": hacs.configuration.dev,
                 "disabled_reason": hacs.system.disabled_reason,
-                "experimental": hacs.configuration.experimental,
```

Comment:
Do the frontend need this?

Reply:
I'm not sure. The intention was to hard code this to True in this PR.

Reply:
No longer needed thanks to https://github.com/hacs/frontend/pull/731